externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/SparkSQLEngine.scala

```diff
@@ -129,7 +129,7 @@ case class SparkSQLEngine(spark: SparkSession) extends Serverable("SparkSQLEngin
         info(s"Spark engine is de-registering from engine discovery space.")
         frontendServices.flatMap(_.discoveryService).foreach(_.stop())
         while (backendService.sessionManager.getOpenSessionCount > 0) {
-          Thread.sleep(1000 * 60)
+          Thread.sleep(TimeUnit.SECONDS.toMillis(5))
```

Comment:
short the sleep interval to pass UT

Reply:
is 10s works well? 5s looks too short and may introduce the noisy logs

Reply:
We can change it just for test, like:

![image](https://github.com/apache/kyuubi/assets/17894939/a320b388-f9d2-4700-b9da-e3ef1667be7f)
![image](https://github.com/apache/kyuubi/assets/17894939/278526ea-8154-498e-9968-91bf55f2b874)

Reply:
done